test/triangulate.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
```

Comment:
since this file doesn't work as intended, I would remove it and place it in a separate branch.